_component_design/Eyebrow.md

```diff
@@ -0,0 +1,430 @@
+
+### Best practice
+
+- The <a href="https://one.rackspace.com/display/reach/Pilot+Shared+Navigation">Pilot navigation API</a> is highly recommended for use in UIs which will be presented along side customer facing applications, such as MyRackspace, FAWS, or the Rackspace Cloud Control Panel.
+- For UIs which will not be integrated with our customer facing product portfolio, the  guidelines in this article can be used to construct eyebrow navigation.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The eyebrow component"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/navigation/eyebrow/eyebrow-hero.png){:width="1200px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Nav bar
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Nav bar specifications
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/navigation/eyebrow/nav-bar.png){:width="936px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+
+## Logo
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Logo structure
+
+The logo can only be clickable when the application has a default landing page, such as a dashboard or overview.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Logo structure when the logo is clickable for navigation"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/navigation/eyebrow/logo-structure.png){:width="620px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+    
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Logo states
+
+- The logo can only be interacted with and clicked when the application has a default landing page, such as a dashboard or overview.
+- If the application has no landing page, then the logo only receives the detault state.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Logo states when the logo is clickable for navigation"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/navigation/eyebrow/logo-states.png){:width="620px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Icon nav item
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Structure
+
+- Nav items fill the height of the nav bar, and have 16 pixels of left and right padding.
```

Comment:
s/`16 pixels`/`16px` for consistency across the site

Reply:
updated.